core/src/main/java/io/questdb/cutlass/text/TextImportExecutionContext.java

```diff
@@ -37,18 +39,26 @@ public class TextImportExecutionContext implements Mutable {
     private final AtomicBooleanCircuitBreaker circuitBreaker = new AtomicBooleanCircuitBreaker();
     // Important assumption: We never access the rnd concurrently, so no need for additional synchronization.
     private final Rnd rnd;
+    private SecurityContext originatorSecurityContext = DenyAllSecurityContext.INSTANCE;
```

Comment:
there are two security contexts when cancelling import. There is one, which originated the import and the other one that invoked "cancel" asynchronously. It is a bit confusing but since there are two, they can't be both referred to as `securityContext`

Reply:
I see. Would be great to add a clarifying comment around this field.